File: Day-06/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hash ={}
        for i, num in enumerate(nums):
            rest = target-num
            if rest in hash:
                return (hash[rest],i)
            hash[num]=i
        
        

File: Day-13.md/twoSum.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        
        HashMap<Integer, Integer> map = new HashMap<>();
        for (int i = 0; i<numbers.length;i++){
            int rest = target - numbers[i];
            if(map.containsKey(rest)){
                return new int[] {map.get(rest)+1,i+1};
            }
            map.put(numbers[i],i);
        }
        return new int[] {};

    }
}
